.circleci/config.yml

```diff
@@ -311,6 +311,17 @@ jobs:
       - run:
           command: yarn && yarn eslint
 
+  typescript-apitests:
+    executor: node/default
+    working_directory: ~/purchases-hybrid-common/typescript
+    steps:
+      - checkout:
+          path: ~/purchases-hybrid-common
```

Comment:
Is this needed?

Reply:
As per https://github.com/RevenueCat/purchases-hybrid-common/pull/455#discussion_r1279118549, we are keeping it like this. We could move the `working_directory` to the steps, but this works for now.